weblate/checks/base.py

```diff
@@ -133,6 +133,17 @@ def check_chars(self, source, target, pos, chars):
 
         return (src in chars) != (tgt in chars)
 
+    def check_interrobang(self, source, target):
+        """Ckeck whether interrobang is present."""
+        strings = ("!?", "?!")
+        try:
+            src = source[-2:]
+            tgt = target[-2:]
+        except IndexError:
+            return False
+
+        return (src in strings) != (tgt in strings)
```

Comment:
I'd move it directly to the `EndInterrobangCheck` class, as it is unlikely this would be ever reused.

```suggestion
```